### PR TITLE
Hot fixes for OS elements

### LINKED ISSUE #27: add multicasting to lo
@neopar it would be cool, if we add it as part of PR #24

### PR DESCRIPTION
**Pull request by padhupradheep**

### CODE REVIEW

File: debian/neobotix-tools/DEBIAN/postinst [hunk 1]
```diff
@@ -42,6 +44,24 @@ sed -i -e 's%GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"%GRUB_CMDLINE_LINUX_DEFAUL
 sed -i -e 's%#GRUB_TERMINAL=console%GRUB_TERMINAL=console%g' /etc/default/grub
 update-grub
 
+# Configure vino
+stty_orig=$(stty -g)
+stty -echo
+IFS= read -p "Please enter VNC password: " -r passwd
+stty "$stty_orig"
+echo ''
+gsettings set org.gnome.Vino require-encryption false
```
Comment (padhupradheep) [inline debian/neobotix-tools/DEBIAN/postinst]: Seems the issue is that these parameters are not set...

Comment (padhupradheep) [reply debian/neobotix-tools/DEBIAN/postinst]: after I manually gave the settings it worked! (Note just the first 2)

**Comment by padhupradheep** [comment]
Also the screen time's out after certain period of inactivity.. but I guess that's okay.. I don't see why it shouldn't or will cause any sorts or issues..

File: debian/neobotix-tools/DEBIAN/postinst (hunk 1 above)
Comment (neopar) [reply debian/neobotix-tools/DEBIAN/postinst]: Yes, just checked it. `require-encryption` and `prompt-enabled` both are set to true. I think this settings need to be set from the defaults user account.

Comment (neopar) [reply debian/neobotix-tools/DEBIAN/postinst]: efd37347b33ce5dcc2f2f2977635f771b1ad99c4 this resolves the issue.

**Comment by neopar** [comment]
> Also the screen time's out after certain period of inactivity.. but I guess that's okay.. I don't see why it shouldn't or will cause any sorts or issues..

271e639a05735a4952994e1af7d44e51b6094811 should do the trick.

File: debian/neobotix-tools/opt/neobotix/tools/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+rm -rf /home/$user/pilot/user/data/logs/*
+
+# delete system logs
+rm -rf /var/log/*
+
+# this is needed for journalctl (systemd)
+mkdir /var/log/journal
+
+# clear bash history
+history -c
```
Comment (padhupradheep) [inline debian/neobotix-tools/opt/neobotix/tools/cleanup.sh]: unfortunately :frowning_face: , history needs to be cleared as just user.. else it complains:

`./cleanup.sh: 29: history: not found`

Comment (neopar) [reply debian/neobotix-tools/opt/neobotix/tools/cleanup.sh]: I did some tests. I was not able to figure out a way to clear current history of the parent session.

Comment (padhupradheep) [reply debian/neobotix-tools/opt/neobotix/tools/cleanup.sh]: the easiest way for this right now would be to remove the `~/.bash_history` found in the home directory. I guess you can do this even with the `sudo command`

`history -c` just removes the history only on the terminal from which we are running the command. But the history would pop again if we open a new terminal once again..

**Pull request merged**